File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1847.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);

        int a = ler.nextInt(), b = ler.nextInt(), c = ler.nextInt();
        boolean feliz=false;
        
        if(a>b && b<=c) feliz=true;
        else if(b>a && c>b && c-b>=b-a) feliz=true;
        else if(a>b && b>c && b-c<a-b) feliz=true;
        else if(a==b && c>b) feliz=true;
        
        if(feliz) System.out.printf(":)\n");
        else System.out.printf(":(\n");
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1847.kt
import java.util.*
import java.lang.Math

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)

    var a = ler.nextInt();
	var b = ler.nextInt();
	var c = ler.nextInt();
    var feliz=false;
    
    if(a>b && b<=c) feliz=true;
    else if(b>a && c>b && c-b>=b-a) feliz=true;
    else if(a>b && b>c && b-c<a-b) feliz=true;
    else if(a==b && c>b) feliz=true;
        
    if(feliz) System.out.printf(":)\n");
    else System.out.printf(":(\n");
}